### ISSUE TITLE
bluetooth is not connecting  and searching any device

### ISSUE THREAD

**Issue by deepakKhetwal20** [elementary/switchboard-plug-bluetooth#83]
bluetooth is not connecting  and searching any device tried multiple times and tried command line also soft block and everthing but its not working

**Comment by marcgm97** [comment]
@deepakKhetwal20 hey I have the same problem, do you have any solution to resolve this problem? thankss

**Comment by deepakKhetwal20** [comment]
No, actually it is a driver issue which we both are facing in legacy hardware, there is no solution to it just move back to window or change the hardware or buy a bluetooth adapter.
